features/steps/task_steps.ts

```diff
@@ -0,0 +1,95 @@
+import {Given, Then, When} from '@cucumber/cucumber';
+import * as Db from '../../src/db';
+import path from 'path';
+import {getRandomId} from '../../src/util';
+import {expect} from 'chai';
+import {readFile} from 'fs/promises';
+import {ServerInjectResponse} from '@hapi/hapi';
+import {testServer} from '../support/hooks';
+
+interface TaskStepsContext {
+    defaultTaskId: string,
+    response: ServerInjectResponse,
+}
+
+
+Given(/^there is a default task in the database$/, async function (this: TaskStepsContext) {
+    const taskId = getRandomId();
```

Comment:
It is usually better not to rely on any randomness in tests (or a least to mock the random function to have a fixed value)